dolphinscheduler-registry/dolphinscheduler-registry-plugins/dolphinscheduler-registry-zookeeper/src/main/java/org/apache/dolphinscheduler/plugin/registry/zookeeper/ZookeeperRegistryProperties.java

```diff
@@ -100,11 +101,17 @@ public static final class ZookeeperProperties {
         private String namespace = "dolphinscheduler";
         private String connectString;
         private RetryPolicy retryPolicy = new RetryPolicy();
-        private String digest;
+        private Authorization authorization = new Authorization();
```

Comment:
```suggestion
        private Map<String, String> authorization = new HashMap<>();
```

Reply:
Use Map is enough, then we don't need know the key in code.

Reply:
> Use Map is enough, then we don't need know the key in code.

Thanks, I will correct this one.